.classpath

```diff
@@ -8,6 +8,7 @@
 	<classpathentry kind="lib" path="lib/commons-configuration-1.9.jar"/>
 	<classpathentry kind="lib" path="lib/commons-httpclient-3.1.jar"/>
 	<classpathentry kind="lib" path="lib/commons-lang-2.6.jar"/>
+	<classpathentry kind="lib" path="lib/commons-io-2.5.jar"/>
```

Comment:
Is this really needed? The tests compile and pass without it (it's already included in a dependency add-on, not the best but not necessary to add it).

Reply:
Intellij refused to build the ("build all") project without this, but if its unnecessary i'll remove it.

Reply:
OK, if IntelliJ needs it we can leave it.